src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -308,10 +308,10 @@
 					x:Name="SettingsButton"
 					AccessKey="I"
 					AutomationProperties.AutomationId="SettingsButton"
-					AutomationProperties.Name="{helpers:ResourceString Name=Settings}"
-					Command="{x:Bind SettingsButtonCommand, Mode=OneWay}"
+					AutomationProperties.Name="{x:Bind ViewModel.Commands.OpenSettings.Label, Mode=OneWay}"
+					Command="{x:Bind ViewModel.Commands.OpenSettings, Mode=OneWay}"
 					Style="{StaticResource AddressToolbarButtonStyle}"
-					ToolTipService.ToolTip="{helpers:ResourceString Name=Settings}"
+					ToolTipService.ToolTip="{x:Bind ViewModel.Commands.OpenSettings.Description, Mode=OneWay}"
```

Comment:
Why is `Mode=OneWay` here? How could it possibly be TwoWay? Is this proper coding?

Reply:
The default is `OneTime`, in any event, the tooltip should probably be the label with hotkey.

Reply:
Because I get `OneTime` as the default and the tooltip shows empty